### PR TITLE
add all the swagger decorators and documentation generation for bulk …

### PR DESCRIPTION
**Pull request by richard-jones**
…applications

### CODE REVIEW

File: portality/view/api_v1.py [hunk 1]
```diff
@@ -256,6 +256,7 @@ def retrieve_journal(journal_id):
 
 @blueprint.route("/bulk/applications", methods=["POST"])
 @api_key_required
+@swag(swag_summary='Create applications in bulk', swag_spec=ApplicationsBulkApi.create_swag())  # must be applied after @api_key_(optional|required) decorators. They don't preserve func attributes.
```
Comment (emanuil-tolev) [inline portality/view/api_v1.py]: we usually add ``` <span class="red">[Authenticated, not public]</span>``` at the end of the route description just to make people aware of what's public at a glance

File: portality/view/api_v1.py [hunk 2]
```diff
@@ -276,6 +277,7 @@ def bulk_application_create():
 
 @blueprint.route("/bulk/applications", methods=["DELETE"])
 @api_key_required
+@swag(swag_summary='Delete applications in bulk', swag_spec=ApplicationsBulkApi.delete_swag())  # must be applied after @api_key_(optional|required) decorators. They don't preserve func attributes.
```
Comment (emanuil-tolev) [inline portality/view/api_v1.py]: ditto

File: portality/view/api_v1.py (hunk 1 above)
Comment (emanuil-tolev) [inline portality/view/api_v1.py]: sorry, to the "swag_summary" I should've said

**Comment by Steven-Eardley** [comment]
I've made the changes ET suggested; merging.

**Pull request merged**